core/Cargo.toml

```diff
@@ -16,7 +16,7 @@ fnv = "1.0"
 futures = { version = "0.3.28", features = ["executor", "thread-pool"] }
 futures-timer = "3"
 instant = "0.1.11"
-libp2p-identity = { version = "0.1", path = "../identity", features = ["peerid", "ed25519"] }
+libp2p-identity = { version = "0.2.3", path = "../identity", features = ["peerid", "ed25519"] }
```

Comment:
We can't make the bump to `0.2` here, that is a breaking change and can't be released as a patch. You'll to need to patch the `0.1` source code of `libp2p-identity`.